datanode/entities/proposal.go

```diff
@@ -173,10 +173,13 @@ func (p Proposal) Cursor() *Cursor {
 }
 
 func (p Proposal) ToProtoEdge(_ ...any) (*v2.GovernanceDataEdge, error) {
-	proposalsProto := make([]*vega.Proposal, len(p.Proposals))
+	j := len(p.Proposals) - 1
+	proposalsProto := make([]*vega.Proposal, j+1)
 
 	for i, proposal := range p.Proposals {
-		proposalsProto[i] = proposal.ToProto()
+		// fill slice in reverse order. The records are inserted (and returned) in reverse order, so to match
```

Comment:
I don't think the issue is that they are backwards? The query returns them ordered by enactment time (see `getProposalsInBatch(ctx context.Context, batchID string)`), whereas FE need them ordered the same as they wre in the original proposal submission.

Reply:
I was wondering if it could really be this simple... clearly not. I'll have a rethink